app/main.py

```diff
@@ -1,3 +1,73 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int) -> object:
```

Comment:
`other` can be float and Distance too. Show it in type hint and fix everywhere
```suggestion
    def __add__(self, other: int | float | Distance) -> object:
```

Reply:
pls check my last commit